modules/internal/ChapelDistribution.chpl

```diff
@@ -581,7 +587,19 @@ module ChapelDistribution {
     // atomics are available
     var _arrAlias: BaseArr;    // reference to base array if an alias
     var pid:int = nullPid; // privatized ID, if privatization is supported
+
+    proc isSliceArrayView() param {
+      return false;
+    }
+
+    proc isRankChangeArrayView() param {
+      return false;
+    }
   
+    proc isReindexArrayView() param {
+      return false;
+    }
```

Comment:
I'm not sure these belong here. Two reasons:
 1) I think it could be a simpler query
 2) I thought we were checking for these with canResolve